app/schema.py

```diff
@@ -51,27 +51,30 @@ class Mentee(BaseModel):
     profile_id: constr(max_length=255)
     first_name: constr(max_length=255)
     last_name: constr(max_length=255)
-    email: constr(max_length=255)
+    email: EmailStr
     country: constr(max_length=255)
     state: constr(max_length=255)
     city: constr(max_length=255)
     formerly_incarcerated: bool
     underrepresented_group: bool
     low_income: bool
-    list_convictions: constr(max_length=255)
+    list_convictions: Optional[constr(max_length=255)]
```

Comment:
This would be better as an array if they have multiple convictions

Reply:
I agree. Can Web send us an array based on the input from the uses?